api/V1/Recording.js

```diff
@@ -181,6 +184,31 @@ module.exports = (app, baseUrl) => {
     })
   );
 
+  /**
+   * @api {get} /api/v1/recordings/report Generate report for a set of recordings
+   * @apiName Report
+   * @apiGroup Recordings
+   * @apiDescription Parameters are as per GET /api/V1/recordings. On
+   * success (status 200), the response body will contain CSV
+   * formatted details of the selected recordings.
+   *
+   * @apiUse V1UserAuthorizationHeader
+   * @apiUse QueryParams
+   * @apiParam {JSON} [tags] As per GET /api/V1/recordings
+   * @apiParam {String} [tagMode] As per GET /api/V1/recordings
```

Comment:
You can use `apiDefine` here.

Reply:
I'll move tags and tagMode into QueryParams